libr/main/rahash2.c

```diff
@@ -403,7 +403,7 @@ static int encrypt_or_decrypt(RahashOptions *ro, const char *hashstr, int hashst
 	const int direction = ro->direction;
 	const char *algo = ro->algorithm;
 	// TODO: generalise this for all non key encoding/decoding.
-	bool no_key_mode = !strcmp ("base64", algo) || !strcmp ("base91", algo) || !strcmp ("punycode", algo);
+	bool no_key_mode = !strcmp ("base64", algo) || !strcmp ("base91", algo) || !strcmp ("punycode", algo) || !strcmp ("bech32", algo);
```

Comment:
this will be fixed after the release. because it requires an abi breaking change, but for now i'm happy to merge it that way. thanks